src/presets.asm

```diff
@@ -568,65 +613,50 @@ org $809AC9
 ; Category Menus/Data
 ; -------------------
 
-org $F18000
-  incsrc presets/prkd_menu.asm
-  incsrc presets/kpdr21_menu.asm
-  incsrc presets/hundo_menu.asm
-  incsrc presets/100early_menu.asm
-  incsrc presets/rbo_menu.asm
-  incsrc presets/pkrd_menu.asm
-  incsrc presets/kpdr25_menu.asm
-  print pc, " preset_menu bankF1 end"
-
-org $F28000
-  incsrc presets/gtclassic_menu.asm
-  incsrc presets/14ice_menu.asm
-  incsrc presets/14speed_menu.asm
-  incsrc presets/allbosskpdr_menu.asm
-  incsrc presets/allbosspkdr_menu.asm
-  incsrc presets/allbossprkd_menu.asm
-  incsrc presets/gtmax_menu.asm
-  incsrc presets/nintendopower_menu.asm
-  incsrc presets/100map_menu.asm
-  print pc, " preset_menu bankF2 end"
-
-org $EF8000
-  incsrc presets/prkd_data.asm
-  print pc, " preset_data bankEF end"
-
-org $EE8000
-  incsrc presets/kpdr21_data.asm
-  incsrc presets/rbo_data.asm
-  print pc, " preset_data bankEE end"
-
-org $ED8000
-  incsrc presets/gtclassic_data.asm
+org $EAE000
+check bankcross off
+print pc, " preset data crossbank start"
```

Comment:
Previously presets started at bank E8.  Now starting near the end of bank EA (after the tile tables which start in bank E8).  Currently the data ends halfway through bank EE leaving bank EF open.  I just added 86K of data and saved a bank instead of using three.

Granted space isn't a big issue on vanilla practice rom yet; if it is we can gut the tile tables.  The best feature here is the lower maintenance.  Simply list all of the preset data together, no need to worry about the individual sizes.

Reply:
Might wanna throw a warnpc on this to keep it out of bank $F0 (infohud.asm). I just tested a build with all of my presets at once and ended up going over. Had to move the tile tables back a bank.

Reply:
There is a warnpc $F08000, it didn't fail the build?

Reply:
I must have just missed it when applying the changes to my branch. Thought I looked for it before posting but I must not have since I replied here instead of adding a new inline comment.